mpd/README.md

```diff
@@ -3,296 +3,518 @@
 ## Requirements
 
 You only need one or more Music Player Daemons (MPD).
+For more information about MPD and it's commands see http://www.musicpd.org/doc/protocol/command_reference.html
 
 ## Configuration
 
-### plugin.conf (deprecated) / plugin.yaml
-
-```
-[mpd]
-    class_name = MPD
-    class_path = plugins.mpd
-```
+### plugin.yaml
 
 ```yaml
 mpd:
     class_name: MPD
     class_path: plugins.mpd
+    instance: "bathroom"
+    host: "192.168.177.164"
+    port: 6601
+#    cycle: 2
+
+mpd:
+    class_name: MPD
+    class_path: plugins.mpd
+    instance: "kitchen"
+    host: "192.168.177.164"
+    port: 6602
+#    cycle: 2
```

Comment:
Usually you don't need to quote all the strings in the YAML configuration files.

Reply:
Are there any exceptions? Some situations where quoting is necessary? I started quoting strings because I ran into problems with an url that was was not quoted in a YAML file.

Reply:
In most cases I don't quote something. Only in special cases where values contains special characters (like ":", "=", "\r", "\n", ..) or in situations producing problems reading the value.

But it's not a big deal to quote it here in the example. It may confuse users seeing some examples with quotes, other without.

Reply:
Ok, that clears up. I will remove the quotes and have a look to your ohter comments during the next week.